### DISCUSSION TITLE
Where to find icon and icon overlay meanings

### DISCUSSION THREAD

**Discussion by dentsu-sh** [dbeaver/dbeaver#16985] [General]
I'm not able to find a documentation, which explains the icons and icon overlay meanings. Especially right now, those in the [database navigator](https://github.com/dbeaver/dbeaver/wiki/Database-Navigator).

![Database-Navigator](https://user-images.githubusercontent.com/108812773/177560840-17018290-0122-4f43-8366-663ee1fdb0f0.png)

What does the black triangle - or what ever it actually is, cause it's so small I cannot identify the shape - in the database icon for the playground_temp_tblsp or postgis-24_sample database mean?

thx.

**Comment by LonwoLonwo** [comment]
Hello @dentsu-sh 

In this case for PostgreSQL `playground_temp_tblsp` and `postgis-24_sample` icons overlay means that these are not default databases. DBeaver opens new connections for non-default PG databases and for this reason non-default databases are marked as special in the database navigator tree.
